agent/fw_drupal.c

```diff
@@ -20,6 +20,8 @@
 #include "util_memory.h"
 #include "util_strings.h"
 
+#define PHP_PACKAGE_NAME "drupal/drupal"
```

Comment:
[fw_drupal8.c](https://github.com/newrelic/newrelic-php-agent/blob/956976b451cd72a433a414d25eca6b9e5f2b06c4/agent/fw_drupal8.c#L21) defines this value as `"drupal/core"`. Is the difference here to use `"drupal/drupal"` intentional?